exporter/signalfxexporter/internal/translation/default_metrics.go

```diff
@@ -118,10 +118,17 @@ exclude_metrics:
   - '!k8s.container.memory_limit'
   - '!k8s.container.cpu_limit'
 
+  # matches all container request metrics but k8s.container.cpu_request and k8s.container.memory_request
   - /^k8s\.container\..+_request$/
+  - '!k8s.container.memory_request'
+  - '!k8s.container.cpu_request'
 
-  # matches any node condition but k8s.node.condition_ready
+  # matches any node condition but memory_pressure, network_unavailable, out_of_disk, p_i_d_pressure, and ready
   - /^k8s\.node\.condition_.+$/
```

Comment:
Same q here

Reply:
Kubernetes has the default node conditions which are well documented and are what we exclude here. Kubernetes built  a [node problem detector](https://github.com/kubernetes/node-problem-detector) you can install that reports several more node conditions. Splunk content has prebuilt filters for the default node conditions.